src/SQLCover/SQLCover/Source/DatabaseSourceGateway.cs

```diff
@@ -173,7 +179,7 @@ private bool DoesNotMatchFilter(string name, List<string> objectFilter, List<str
 
             foreach (var filter in excludedObjects)
             {
-                if (filter == lowerName)
+                if (filter == lowerName || filter == name)
```

Comment:
You could alternatively use `string.Compare(filter, name, StringComparison.InvariantCultureIgnoreCase) == 0` and get rremove `lowerName`.

Reply:
Thank you for pointing it out, as they say we learn every day!